central/sac/authorizer/builtin_scoped_authorizer_test.go

```diff
@@ -93,13 +93,6 @@ func TestBuiltInScopeAuthorizerWithTracing(t *testing.T) {
 			scopeKeys: readCluster(firstClusterID, resources.Compliance.Resource),
 			results:   []bool{false, false, true},
 		},
-		{
-			name:  "allow read from compliance with replacing resource permissions",
-			roles: []permissions.ResolvedRole{role(complianceEdit, withAccessTo1Cluster())}, // TODO: ROX-12750 Replace ComplianceRuns with Compliance.
-
-			scopeKeys: readCluster(firstClusterID, resources.ComplianceRuns.Resource),
-			results:   []bool{false, false, true},
-		},
```

Comment:
If the replacing resource concept is to be kept, it might be good to have two internal test resources that allow tests to cover the cases of replaced resource access checks pass when access granted to replacing resource.